### PR TITLE
Move ci builds to sonarqube team gcp project.

### PR DESCRIPTION
**Pull request by malena-ebert-sonarsource**
Do not merge

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -32,9 +32,9 @@ env:
 # RE-USABLE CONFIGS
 #
 container_definition: &CONTAINER_DEFINITION
-  image: gcr.io/ci-cd-215716/base:latest
-  cluster_name: cirrus-uscentral1a-cluster
-  zone: us-central1-a
+  image: gcr.io/sonarqube-team/base:latest
+  cluster_name: cirrus-ci-cluster
```
Comment (tomverin) [inline .cirrus.yml:36]: Your cluster name is to generic, should have use something like cirrus-ci-sq-cluster

File: .cirrus.yml [hunk 2]
```diff
@@ -32,9 +32,9 @@ env:
...
+  zone: europe-west3-a
```
Comment (wouter-admiraal-sonarsource) [inline .cirrus.yml:37]: Go 🇪🇺 !

File: .cirrus.yml [hunk 3]
```diff
@@ -66,10 +66,10 @@ linux_qa_task:
   <<: *ONLY_SONARSOURCE_QA
   gke_container:
     dockerfile: it/docker/Dockerfile
-    builder_image_project: ci-cd-215716
+    builder_image_project: sonarqube-team
```
Comment (wouter-admiraal-sonarsource) [inline .cirrus.yml:69]: Out of curiosity: is this a brand new GCE Project? Or you renamed the existing "ci-cd-215716" one? If the former, I guess we will move all other projects, too? If the latter... I guess everything would start breaking down, so it seems unlikely 😅 .

File: .cirrus.yml (hunk 1 above)
Comment (malena-ebert-sonarsource) [reply .cirrus.yml:36]: It is the `cirrus-ci-cluster` within the `sonarqube-team` project.

File: .cirrus.yml (hunk 3 above)
Comment (malena-ebert-sonarsource) [reply .cirrus.yml:69]: It is a brand new one.
I will move every project which belongs to the SQ team to new GCP project.

**Pull request merged**